pptx_generator/templates.py

```
@@ -1,6 +1,16 @@
 from pptx.util import Inches, Pt
+from pptx.dml.color import RGBColor
```

Ruff (0.12.2)

`pptx.dml.color.RGBColor` imported but unused

Remove unused import: `pptx.dml.color.RGBColor`

(F401)